heatmap/pod-network-latency.html

```diff
@@ -1,35 +1,60 @@
 <html>
-<head>
-<style>
-h1 {text-align: center;color:#339933;}
-h3 {text-align: center;color:#339933;}
-p {text-align: center;}
-div {text-align: center;}
-.aligncenter {
-    text-align: center;
-}
-</style>
-</head>
-<body>
-
-<h1><u>Pod Network Latency Chaos</u></h1>
-<br>
-<p>Pod Network Latency network latency into application pod. Here are the different coverage metrics for the experiment.
-<br><br>
-
-<h3><i> Pod Network Latency experiment based coverage<i></h3>
-<div>
-<img src="../images/pod-network-latency-heatmap.png" class="aligncenter" alt="Pod Network Latency Heatmap coverage" width="1100" height="700">
-</div>
-
-<h3><i> Pod Network Latency cluster runtime based coverage<i></h3>
-<div>
-<img src="../images/pod-network-latency-runtime-heatmap.png" class="aligncenter" alt="Pod Network Latency Runtime Heatmap coverage" width="1000" height="500">
-</div>
-<h3><i> Pod Network Latency platform based coverage<i></h3>
-<div>
-<img src="../images/pod-network-latency-platform-heatmap.png" class="aligncenter" alt="Pod Network Latency Platform Heatmap coverage" width="1100" height="700">
-</div>
-</p>
-</body>
-</html>
+  <head>
+    <link rel="stylesheet" href="mystyle.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type = "text/javascript" src="function.js">
+      google.charts.load('current', {packages: ['corechart', 'bar']});
+      google.charts.load('current', {'packages':['table']});
+    </script>
+  </head>
+  <body>
+    <h1><u>Pod Network Latency Chaos</u></h1>
+    <br>
+    <p> Pod Network Latency platform tested</p>
+    <br>
+    <div style="width: 1000px; text-align: center; margin: 0 auto;">
+      <img src="../images/platforms/GKE.png" alt="HTML5 Icon" style="width:60px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/EKS.png" alt="HTML5 Icon" style="width:140px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/AKS.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kind.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/minikube.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kubeadm.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+    </div>
+    <br><br>
+    <h1><u>Pod Network Latency Experiment Details</u></h1>
+    <p>Pod Network Latency network latency into application pod. Here are the different experiment details.</p>
```

Comment:
```suggestion
    <p>Pod Network Latency injects network latency into application pod. Here are the different experiment details.</p>
```
Can we have this minor change